.github/contributing.md

```diff
@@ -0,0 +1,24 @@
+# How to properly make a request in Minehut Meta
+
+So, you want to suggest a feature, report a bug, or maybe suggest a change for the filter? Always remember to follow these guidelines: 
+
+1. Use the issue templates provided
+2. Help out staff if asked
+3. Contribute to on-going discussions
+4. Give constructive feedback to ideas
+5. Search the issue tracker to ensure that your issue has not already been submitted before making a new post (use the search bar on the Issues page)
+6. Verify the validity of your issue
+7. Be clear, concise and specific in your messages
+8. Keep conversations on-topic
+9. Be respectful toward everyone
+10. Take advantage of GitHub Flavoured Markdown -- it's awesome, and it makes your submission that much better! 😄
+11. Create a separate issue for each feature request/bug report (don't talk about multiple things in one issue)
+
+
+1. Don't bump issues for the sole purpose of receiving a response (all issues are acknowledged)
+2. Don't argue unproductively in the comments
+3. Don't stray off-topic
+4. Don't try to get support for Minehut services here -- read this
+5. Don't use logical fallacies in arguments
```

Comment:
this seems awk here, can we find a better way to phrase this

Reply:
Which line?